plugins/org.locationtech.udig.catalog.wmt/src/org/locationtech/udig/catalog/internal/wmt/ww/WWService.java

```diff
@@ -1,5 +1,4 @@
-/**
- * uDig - User Friendly Desktop Internet GIS client
+/* uDig - User Friendly Desktop Internet GIS client
```

Comment:
wondering about change from "/**" to "/*" for class comment. Have the formatter changed it automatically? if so we need to adapt the formatter or the template ..

Reply:
I reverted one commit so this file is not changed within this PR at all.